connector/src/main/java/com/google/cloud/spark/bigquery/BigQueryDataSourceV2.java

```diff
@@ -15,21 +15,46 @@
  */
 package com.google.cloud.spark.bigquery;
 
+import com.google.cloud.bigquery.connector.common.BigQueryClientModule;
+import com.google.common.collect.ImmutableMap;
+import com.google.inject.Guice;
+import com.google.inject.Injector;
+import org.apache.spark.sql.SparkSession;
 import org.apache.spark.sql.sources.v2.DataSourceOptions;
 import org.apache.spark.sql.sources.v2.DataSourceV2;
 import org.apache.spark.sql.sources.v2.ReadSupport;
 import org.apache.spark.sql.sources.v2.reader.DataSourceReader;
 import org.apache.spark.sql.types.StructType;
 
+import java.util.Optional;
+
+import static scala.collection.JavaConversions.mapAsJavaMap;
```

Comment:
sort imports?

Reply:
They are, based on intellij - static import at the end, java runtime after libraries. We will apply the [Google java code formatter](https://github.com/google/google-java-format) when everything is merged to master.